game-core/src/main/java/games/strategy/engine/xml/TestAttachment.java

```diff
@@ -56,14 +56,14 @@ public void resetValue() {
   public void validate(final GameData data) {}
 
 
-  private Map<String, AttachmentProperty<?>> createPropertyMap() {
-    return ImmutableMap.<String, AttachmentProperty<?>>builder()
-        .put("value", AttachmentProperty.of(this::setValue, this::getValue, this::resetValue))
+  private Map<String, ModifiableProperty<?>> createPropertyMap() {
+    return ImmutableMap.<String, ModifiableProperty<?>>builder()
+        .put("value", ModifiableProperty.of(this::setValue, this::getValue, this::resetValue))
         .build();
   }
 
   @Override
-  public Map<String, AttachmentProperty<?>> getPropertyMap() {
+  public Map<String, ModifiableProperty<?>> getPropertyMap() {
     return createPropertyMap();
```

Comment:
I meant to ask this in the previous PR, but why not just inline the `createPropertyMap()` methods in `getPropertyMap()`?

Reply:
I just did.